http/handler_acp.go

```diff
@@ -113,6 +142,22 @@ func (h *acpHandler) bindRoutes(router *Router) {
 		Value: acpAddDocActorRelationshipRequest,
 	}
 
+	acpDeleteDocActorRelationshipRequest := openapi3.NewRequestBody().
+		WithRequired(true).
+		WithContent(openapi3.NewContentWithSchema(openapi3.NewStringSchema(), []string{"text/plain"}))
```

Comment:
todo: This looks incorrect, you should be strongly typing the schema so that the openAPI magic can document it properly.  Have a look at `openApiSchemas` in http/openAPI.go

Please test this manually after correcting.

Reply:
Resolved in commit: `PR(ADHERE-REVIEW): Fix the open api schemas`